packages/headless/src/features/analytics/analytics-utils.ts

```diff
@@ -242,7 +242,7 @@ export type AnalyticsActionOptions<
   LegacyProvider,
   Client,
   PayloadType,
-> = Exclude<
+> = Omit<
```

Comment:
What is this change for 🤔 ?

Reply:
That was a fix to a problem we had with the `makeAnalyticsActionFactory`, @louis-bompart  identified and fixed this issue by making this change.

Reply:
Yes. oopsie daisy.